docs/software/spack.md

```diff
@@ -0,0 +1,179 @@
+# Spack
+
+*Currently, Spack is only available on Alpine.* [Spack](https://spack.readthedocs.io/en/latest/) is a package manager designed for installing multiple configurations. Spack is designed for Supercomputing clusters, making it easy to install and configure software that may not currently be available in the Alpine software stack. The following documentation demonstrates how to activate & use the CURC Spack module and utilize your own Spack environments to install packages and compilers.
+
+## Using Spack on CURC
+You can follow these steps from a Research Computing terminal session via a compile job (`acompile`) or within a batch or interactive job.
+
+### Activate and Use the CURC Spack Module
+
+Run the following command to load the Spack module:  
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ module load spack/0.20.1
```

Comment:
```suggestion
[johndoe@c3cpu-c11-u17-2 ~]$ module load spack
```

I recommend the following. I think we should stop using specific versions in our documentation, unless we are demonstrating something where is makes sense. By adding versions, it makes the documentation harder to maintain.